### ISSUE TITLE
Does not work on bungeecoord with 1.19 support

### ISSUE THREAD

**Issue by fabegalo** [dejvokep/secured-network#11]
Does not work on bungeecoord with 1.19 support

**Comment by meowsome** [comment]
Happens for me too. Came from BungeeGuard, which has the same issue. Here's a snippet of the issue below (edited to take out personal data):

```
[SecuredNetwork] DENIED (code B4): Failed to authenticate "uuid": passphrase not found. If that was you and you believe this is an error, please try checking for incompatible plugins installed. Data: data[{"name":"textures","value":"value","signature":"signature"}]
```

**Comment by dejvokep** [comment]
Hey 👋

thanks for reporting the issue. I managed to fix it quite quickly, and just now I've released the new update 3.5. Feel free to check it out. In case of any further questions, do not hesitate and contact me again.

Thanks for your time, have an awesome day.
- David
